src/Tribe/Tickets.php

```diff
@@ -1514,12 +1545,18 @@ public static function get_event_attendees_count( $post_id, $args = [] ) {
 		 * @return array
 		 */
 		public static function get_all_event_tickets( $post_id ) {
-			$cache_key = self::$cache_key_prefix . $post_id;
-			$cache = new Tribe__Cache();
-			$tickets = $cache->get( $cache_key );
+			$key = __METHOD__ . '-' . $post_id;
+
+			/* @var Tribe__Cache $cache */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: f42f8934e574852394355c5fc120d75c</sub>